test/test_meta.py

```diff
@@ -185,7 +185,26 @@ def test_tensor_outlives_converter(self):
         del m
         self.assertIs(ref(), None)
 
-def assert_ref_meta_equal(test_case, meta_rs, rs, msg_callable):
+CHECK_STRIDES = {
+    torch.Tensor.__getitem__,
+}
+
+def should_check_strides(func):
+    if func in CHECK_STRIDES:
+        return True
+    if not isinstance(func, torch._ops.OpOverload):
+        return False
+    # Prims are expected to model strides correctly
+    if hasattr(func, 'namespace') and func.namespace == "prims":
```

Comment:
I'm adding this check: `hasattr(func, 'namespace')`
Is it okay or do you want to do something else to avoid an AttributeError?

@ezyang

Reply:
You should revert the change here now...

Reply:
WDYM, I should revert the `def namespace(self):` logic?
I'm not seeing it landed in master, though

Reply:
You should put that change in this PR...

Reply:
Oh yeah, I did :) that's why it's passing now :)

Reply:
And so now you no longer need to test if hasattr namespace...

Reply:
waiittt, I removed that as well :'( Looks like I didn't push it.... yeah thank you for spotting it

Reply:
![image](https://user-images.githubusercontent.com/5086322/191811539-dc1e462d-bcd8-4e39-8055-b5775744fa34.png)
Lmao :'(

Sorry about that